### DISCUSSION TITLE
Question sur les icones

### DISCUSSION THREAD

**Discussion by Mathis554** [witsa/synapps#780] [Q&A] [answered]
Bonjours,
Quelle est le site ou on trouve les icônes de Synapps?
Peux t'on rajouter des icônes dans Synapps?

**Comment by WITNicolas** [comment, marked as answer]
Bonjour,

C'est cette biblioteque qui est intégrée dans STUDIO https://getbootstrap.com/docs/3.3/components/
On ne peut rajouter d'icone au sens propre du terme.

Par contre vous pouvez mettre vos propres images dans la librairie de STUDIO

![image](https://github.com/user-attachments/assets/f1f983fa-2c0f-4eae-a67d-5ca184166528)

**Reply by Mathis554** [reply]
Merci beaucoup c'est parfait comme système en plus on peut rajouter des gifs
